### ISSUE TITLE
Support for Kubernetes 1.29+

### ISSUE THREAD

**Issue by chuchynz** [Azure/aks-engine-azurestack#259]
Will there be support for Kubernetes 1.29 / 1.30?

**Explain why AKS Engine needs it**
AKS Engine should support the current Kubernetes release schedule.

**Additional context**
We deploy both Azure AKS and AKS Engine Kubernetes clusters and would like to ensure we have version parity between these systems, and to keep our clusters as close to the [AKS Release schedule](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar) as possible. 1.30 would be the sweet spot right now.

**Comment by SummerSmith** [comment]
Yes, we will be adding support for Kubernetes 1.29 and 1.30. At this time, I am not able to share specific timelines.

**Comment by ThomasRikardsen** [comment]
When would you be in a position to share timelines on this?

**Comment by mekki209** [comment]
Is there any update about when the new release will be ?

**Comment by haofan-ms** [comment]
k8s 1.29 and 1.30 supported in AKS Engine release v0.82.1
